server/asset/eth/common.go

```diff
@@ -74,6 +74,8 @@ const (
 	// TODO: When the contract is solidified, break down evm functions
 	// called and the gas used for each. (◍•﹏•)
 	InitGas = 180000 // gas
+
+	RedeemGas = 63000
```

Comment:
`// gas` at the end for clarity I guess. Lots of different units going around so best to be precise imo.